### PR TITLE
[IP-695] Improve the pagination of item tables

### PR DESCRIPTION
**Pull request by daniel89fg**
Pull Request Checklist

  * [x] My code follows the code formatting guidelines.
  * [x] I have an issue ID for this pull request. 
  * [x] I selected the corresponding branch.
  * [ ] I have rebased my changes on top of the corresponding branch.
  
Issue Type (Please check one or more)

  * [ ] Bugfix
  * [ ] Improvement of an existing Feature 
  * [x] New Feature

### CODE REVIEW

File: application/core/MY_Model.php [hunk 1]
```diff
@@ -146,6 +146,8 @@ public function paginate($base_url, $offset = 0, $uri_segment = 3)
 
         $this->set_defaults();
         $this->run_filters();
+		
+		$this->current_page = ($offset / $per_page) + 1;
```
Comment (Kovah) [inline application/core/MY_Model.php]: Could you correct your indentation for all files? :)

File: assets/core/css/custom.css [hunk 2]
```diff
@@ -3,4 +3,5 @@
    Custom Stylesheet
 
    All changes made in this file will override the default styles.
-   ========================================================================== */
\ No newline at end of file
+   ========================================================================== */
+   .model-pager-detailed{text-align: center;}
```
Comment (Kovah) [inline assets/core/css/custom.css]: Whoops, no CSS into the custom.css please.

File: application/modules/payments/views/online_logs.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<div id="headerbar">
```
Comment (Kovah) [inline application/modules/payments/views/online_logs.php]: Where does that file come from? Seems to be incorrect, git shows conflicts wit this file.

File: application/helpers/pager_helper.php [hunk 4]
```diff
@@ -43,3 +43,37 @@ function pager($base_url, $model)
...
+    $total_rows = $CI->$model->total_rows;
+    
+    if ($total_rows == 0) {
+        $current_item_last = 0;
+        $current_item_first = 0;
+    }
+    
+    $pager = '<div class="model-pager-detailed">';
+    $pager .= '<div>';
+    $pager .= trans('showing').' '.$current_item_first;
```
Comment (Kovah) [inline application/helpers/pager_helper.php]: You may add actual buttons for navigation here so the user does not have to scroll up to get to the next page

File: assets/core/css/custom.css (hunk 2 above)
Comment (daniel89fg) [reply assets/core/css/custom.css]: But if I add in the css of the core, I have to eliminate the gitignore to be able to synchronize it. Or how is it done?

File: application/modules/payments/views/online_logs.php (hunk 3 above)
Comment (daniel89fg) [reply application/modules/payments/views/online_logs.php]: The last version 1.5.9 exists this file, I just copied it and added the paging line.

File: application/helpers/pager_helper.php (hunk 4 above)
Comment (daniel89fg) [reply application/helpers/pager_helper.php]: It was not my initial idea. Only show information of the total of records and the existence of pages.

File: application/core/MY_Model.php (hunk 1 above)
Comment (daniel89fg) [reply application/core/MY_Model.php]: ready

File: assets/core/css/custom.css (hunk 2 above)
Comment (Kovah) [reply assets/core/css/custom.css]: Just remove it, I will add it to the corresponding file then.

Comment (daniel89fg) [reply assets/core/css/custom.css]: ok, what would be the correct way to do it ?, what file ?, for the next time.

**Comment by UnderDogg** [comment]
Needs label : 1.6 or 1.5.11 or something. Needs Milestone?

**Pull request closed without merging**